config/200-aggregation-admin-clusterroles.yaml

```diff
@@ -0,0 +1,170 @@
+# Copyright 2022 TriggerMesh Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# All sources used at TriggerMesh
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: triggermesh-sources-admin
+  labels:
+    rbac.triggermesh.io/sources-admin: "true"
+rules:
+- apiGroups:
+  - sources.knative.dev
+  - sources.eventing.knative.dev
+  - sources.eventing.triggermesh.dev
```

Comment:
I don't think we have such group in tm/tm.

Reply:
Hmmm thought we did for the early sources work. Needless to say, there's quite a bit of truing up that needs to happen internally.

Reply:
For the record, it is/was only used by an internal source that has since been archived.